packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js

```diff
@@ -41,7 +40,6 @@ describe('normalize2.mod-mods-vals', () => {
         const decl = { mods: { mod: 'val' } };
 
         expect(normalize(decl, scope).map(simplifyCell)).to.deep.equal([
-            { entity: { block: 'sb' }, tech: null },
```

Comment:
Is it wrong? Isn't `normalize({ block: 'b', mods: { m: 'v' })` same as `normalize({ mods: { m: 'v' }), scope({ block: 'b' }))` ?

Reply:
yep, it's wrong